db/migrate/20220331164557_create_transfer_packages.rb

```diff
@@ -0,0 +1,19 @@
+class CreateTransferPackages < ActiveRecord::Migration
+  def up
+    create_table :transfer_packages do |t|
+      t.belongs_to :receiver_institution,            null: false
+      t.string     :uuid,                 limit: 36, null: false
+      t.string     :recipient
+      t.boolean    :includes_qc_info, default: false
```

Comment:
We wouldn't need to persist the value with AR5, where we could just have a virtual boolean attribute on the model:

```ruby
class TransferPackage
  attribute :includes_qc_info, :boolean, default: false
end
```